Classes/Wizard/CoordinatepickerWizard.php

```diff
@@ -1,5 +1,6 @@
 <?php
-declare(strict_types = 1);
+
+declare(strict_types=1);
```

Comment:
Removing white space? Better readable with

Reply:
The core always uses the format `declare(strict_types=1);`